intersection_server: 
  {
    distance_to_intersection: 0.75,
    road_width: 1.05,
    point_min_y: -0.5,
    point_max_y: 0.7,
    stop_time: 3,
    speed_default: 1.0,
    num_corners_to_detect: 1,
    distance_of_blind_approaching: 0.2,
  }